src/features/metrics/14-kube-state-metrics-deployment.yml.hb

```diff
@@ -28,6 +28,11 @@ spec:
                   operator: In
                   values:
                   - linux
+              - matchExpressions:
+                - key: beta.kubernetes.io/arch
+                  operator: In
+                  values:
+                  - amd64
```

Comment:
As you see with the OS selector, we define both versions of the label as separate term items. This is because the terms are OR'ed. So to cover both possible cases, add also the `kubernetes.io/arch` as separate `matchExpressions ` entry.

Reply:
Ah, right.  updated with change to support both labels.